luigi-tutorials/tutorial-container.md

```diff
@@ -0,0 +1,294 @@
+
+Normally, Luigi consists of two main parts: **Luigi Core** and **Luigi Client**. Core refers to the main app which houses the micro frontends, while Client refers to the micro frontends themselves. 
+
+However, a third feature called **Luigi Container** allows you to easily insert Luigi micro frontends anywhere without the need for a Luigi Core application. This feature simplifies the use of Luigi for developers, eliminating the need for significant changes in their applications.
+
+This tutorial will show you how to incorporate micro frontends inside a [UI5](https://sapui5.hana.ondemand.com/sdk/#/) application. Nevertheless, a similar process will also apply to other frontend frameworks such as [Angular](https://angular.io/) or [React](https://react.dev/).
+
+> **NOTE:** In addition to this tutorial, you can also experiment with the Luigi Container test app on [GitHub](https://github.com/SAP/luigi/tree/main/container/test-app).
+
+---
+
+## Steps
+
+### Install UI5 generator 
+
+1. Open a new command prompt/Terminal window and install the UI5 generator: 
+
+`npm install -g yo generator-easy-ui5`
+
+2. Run this command to ensure that Yeoman has been installed correctly: 
+
+`yo`
+
+Make sure the easy-ui5 generator is listed.
+
+### Create UI5 application 
+
+1. Create your UI5 project:
+
+`yo easy-ui5`
+
+2. Answer the prompts in this way to create your new project: 
+
+    <!-- border -->![UI5 Generator](ui5.png)
+
+3. Run the app locally:
+
+```shell
+cd luigi.ui5app
+npm start # or "yarn start"
+```
+
+### Install Luigi Container 
+
+Luigi Container can be installed via a [npm](https://www.npmjs.com/) package.
+
+To use npm packages in UI5, you need to first install the UI5 Tooling.
+
+1. In your project directory, run: 
+
+```shell
+npm i ui5-tooling-modules -D
+```
+
+
+2. In this step, you will add the UI5 tooling task and middleware declaration. Open your application's `ui5.yaml` file. Replace the content with: 
+
+```yaml
+specVersion: "3.0"
+metadata:
+  name: luigi.ui5app
+type: application
+framework:
+  name: OpenUI5
+  version: "1.118.0"
+  libraries:
+    - name: sap.m
+    - name: sap.ui.core
+    - name: themelib_sap_fiori_3
+    - name: themelib_sap_horizon
+builder:
+  customTasks:
+    - name: ui5-tooling-modules-task
+      afterTask: generateVersionInfo
+server:
+  customMiddleware:
+    - name: ui5-tooling-modules-middleware
+      afterMiddleware: compression
+    - name: ui5-middleware-livereload
+      afterMiddleware: compression
+```
+
+3. Create a new folder called `control` in `luigi.ui5app/webapp`. In it, create a file called `LuigiContainer.js` and paste the following content into it: 
+
+```javascript
+HTMLElement.prototype.attachInvalidate = function() {};
```

Comment:
Is this still needed? Maybe we still need it until the 1.120.0 is released which includes the fix for this.

Reply:
The tutorial still works without this line, so I omitted it 😄